be/src/util/rle_encoding.h

```diff
@@ -451,7 +451,7 @@ inline size_t RleDecoder<T>::GetBatch(T* vals, size_t batch_num) {
 // it decides whether they should be encoded as a literal or repeated run.
 template <typename T>
 inline void RleEncoder<T>::Put(T value, size_t run_length) {
-    DCHECK(bit_width_ == 64 || value < (1LL << bit_width_));
```

Comment:
If the bit_width_ is 64, 1<<64 will be overflow, and the check will be false.
So we need some checks before the left shift operation

Reply:
1LL is large int